src/arrayMethodSort.js

```diff
@@ -4,9 +4,42 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
+  [].__proto__.sort2 = function(compareFunction = compareStrings) {
     // write code here
+    let counter;
+
+    do {
+      counter = 0;
+
+      for (let i = 0; i < this.length - 1; i++) {
+        const a = this[i];
+        const b = this[i + 1];
+
+        if (compareFunction(a, b) > 0) {
+          this[i] = b;
+          this[i + 1] = a;
+          counter++;
+        }
+      }
+    } while (counter > 0);
+
+    return this;
   };
 }
 
+function compareStrings(a, b) {
```

Comment:
the definition of this function should probably be placed inside the sort2() method, so that the method does not depend on any external functions